Osiris/Hacks/Visuals.cpp

```diff
@@ -406,3 +406,32 @@ void Visuals::skybox(FrameStage stage) noexcept
         memory->loadSky(sv_skyname->string);
     }
 }
+
+void Visuals::xCrosshair() noexcept
+{
+    static auto crosshairalpha = interfaces->cvar->findVar("cl_crosshairalpha");
```

Comment:
Maybe some people use a crosshair with less than 255 alpha, why don't you use the cvar "crosshair" instead?

Reply:
10000in1 lol

Reply:
thats 1 in 10000 situation lol u can configure the code too. im not saying mine is clearest one 
just 5 minute stuff cuz someone asked